src/Fixer/PhpTag/EchoTagSyntaxFixer.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\PhpTag;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+/**
+ * @author Michele Locati <michele@locati.it>
+ */
+final class EchoTagSyntaxFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /** @internal */
+    const OPTION_FORMAT = 'format';
+
+    /** @internal */
+    const OPTION_SHORT_ALWAYS = 'short_always';
+
+    /** @internal */
+    const OPTION_LONG_FUNCTION = 'long_function';
+
+    /** @internal */
+    const FORMAT_SHORT = 'short';
+
+    /** @internal */
+    const FORMAT_LONG = 'long';
+
+    /** @internal */
+    const LONG_FUNCTION_ECHO = 'echo';
+
+    /** @internal */
+    const LONG_FUNCTION_PRINT = 'print';
+
+    private $supportedFormatOptions = [
```

Comment:
Can be constant as well (same for `$supportedLongFunctionOptions`) - it is not changing during the run.

Reply:
Done in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4493/commits/034546b04580b786f795945c9f191dfc9cb8e218